src/Rector/ClassMethod/ConsoleExecuteReturnIntRector.php

```diff
@@ -229,6 +230,9 @@ private function setReturnTo0InsteadOfNull(Return_ $return): void
         }
 
         $staticType = $this->getType($return->expr);
+        if ($staticType instanceof ErrorType) {
+            return;
+        }
```

Comment:
this can be removed as now uses correct constant

Reply:
Does it make sens to cast to int an unknown constant ?

I would have say it was ok to keep this check.

Reply:
Oh, it seems you removed the ErrorType check 👍  https://github.com/rectorphp/rector-symfony/pull/296/commits/b4ede768547e400d131f66219c4bf2ac25749b83, we can verify in the future if it actually needed or not to verify ErrorType :)

Reply:
Yes, we will need to take a look at https://getrector.org/demo/5359325c-f447-4c15-b703-31ba2ec5ddee when the sandbox will be updated

Reply:
that will be updated in next 4 hours.